guvnor-ala/guvnor-ala-spi/src/main/java/org/guvnor/ala/registry/SourceRegistry.java

```diff
@@ -34,8 +33,8 @@
      * @param Repository meta data
      * @see Repository
      */
-    void registerRepositorySources( final Path path,
-            final Repository repo );
+    void registerRepositorySources(final Path path,
```

Comment:
I think you've used the wrong formatting for methods in this file (they should have spaces around arguments, right?).